lib/profile/sections/contact.dart

```diff
@@ -0,0 +1,162 @@
+    return Container(
+      width: MediaQuery.of(context).size.width,
+      padding: EdgeInsets.fromLTRB(30.0, 10.0, 30.0, 20.0),
+      decoration: BoxDecoration(
+        border: Border(
+          bottom: BorderSide(
+            width: 5.0,
+            color: Colors.black12
+          )
+        )
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          SectionHeader(title: SECTION_TITLE_CONTACT),
+          buildInfoRow(
+            context, 
+            email, 
+            CONTACT_EMAIL_ADDRESS, 
+            contact["emailAddress"],
+            handleEmailTap
+          ),
+          buildInfoRow(
+            context,
+            phone,
+            CONTACT_PHONE_NUMBER,
+            contact["phoneNumber"],
+            handlePhoneTap
+          ),
+          buildInfoRow(
+            context,
+            web,
+            CONTACT_WEBSITE,
+            contact["website"],
+            handleWebsiteTap
+          )
+        ],
+      )
+    );
+  }
+
+  buildInfoRow(
+    BuildContext context, 
+    IconData iconName, 
+    String label, 
+    String info, 
+    Function() handleTap
+  ) {
+    return InkWell(
+      onTap: handleTap,
+      child: Container(
+        width: MediaQuery.of(context).size.width,
+        padding: EdgeInsets.fromLTRB(0.0, 10.0, 0.0, 10.0),
+        decoration: BoxDecoration(
+          border: Border(
+            bottom: BorderSide(
+              color: Colors.black12
+            )
+          )
+        ),
+        child: Row(
+          children: <Widget> [
+            Container(
+              padding: EdgeInsets.fromLTRB(10.0, 0.0, 10.0, 0.0),
+              child: Icon(
+                iconName,
+                color: Colors.black45,
+                size: 25.0,
+              )
+            ),
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                buildLabel(label),
+                buildInfo(info)
+              ]
+            )
+          ]
+        )
+      )
+    );
+  }
+
+  buildLabel(String label) {
+    return Text(
+      label,
+      textAlign: TextAlign.left,
+      style: GoogleFonts.muli(
+        textStyle: TextStyle(
+          color: Colors.black,
+          letterSpacing: .5,
+          height: 1.2,
+          fontSize: 13.0,
+          fontWeight: FontWeight.w700
+        ),
+      )
+    );
+  }
+
+  buildInfo(String info) {
+    return Text(
+      info,
+      textAlign: TextAlign.left,
+      style: GoogleFonts.muli(
+        textStyle: TextStyle(
+          color: Colors.black54,
+          letterSpacing: .5,
+          height: 1.2,
+          fontSize: 13.0,
+          fontWeight: FontWeight.w500
+        ),
+      )
+    );
+  }
+
+  handleEmailTap() {
+    _launchUrl("mailto:" + contact["emailAddress"]);
+  }
+
+  handlePhoneTap() {
+    _launchUrl("tel:" + contact["phoneNumber"]);
+  }
+
```

Comment:
Like I said in the very top comment, iOS simulators do not have mail and dialer apps installed so these two handlers will not redirect you anywhere if clicked. To test it, you'd have to do it on an iOS device. Here are the steps that I followed to run the app on my iPhone: https://medium.com/front-end-weekly/how-to-test-your-flutter-ios-app-on-your-ios-device-75924bfd75a8